apps/InfiniteRedAI/app/screens/DocumentScannerScreen.android.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { FC } from "react"
+import { observer } from "mobx-react-lite"
+import { ViewStyle, View, ImageStyle, TextStyle } from "react-native"
+import { NativeStackScreenProps } from "@react-navigation/native-stack"
+import { AppStackScreenProps } from "../navigators"
+import { Screen, Text, Icon, Button, ListItem, TextField, Toggle } from "../components"
+import { useTypedNavigation } from "../navigators/useTypedNavigation"
+import {
+  ResultFormatOptions,
+  ScannerModeOptions,
+  launchDocumentScannerAsync,
+} from "@infinitered/react-native-mlkit-document-scanner"
+import { spacing } from "app/theme"
+
+interface DocumentScannerScreenProps
+  extends NativeStackScreenProps<AppStackScreenProps<"DocumentScanner">> {}
+
+export const DocumentScannerScreen: FC<DocumentScannerScreenProps> = observer(
+  function DocumentScannerScreen() {
+    const navigation = useTypedNavigation<"ImageLabeling">()
```

Comment:
Can you update this other missed part, `useTypedNavigation<"ImageLabeling">()` => `useTypedNavigation<"DocumentScanner">()?